### PR TITLE
Added cucumber-tsflow

### PR DESCRIPTION
**Pull request by DennisJaamann**

### CODE REVIEW

File: test/e2e/authentication/authentication.page.ts [hunk 1]
```diff
@@ -13,27 +13,27 @@ export class AuthenticationPageObject {
 
   }
 
-  openApp(){
+  public openApp(): webdriver.promise.Promise<void> {
```
Comment (samvloeberghs) [inline test/e2e/authentication/authentication.page.ts]: methods are public by default, you can safely ommit the keyword

File: test/e2e/authentication/login/login.page.ts [hunk 2]
```diff
@@ -20,40 +23,33 @@ export class LoginPageObject {
 
   }
 
-  navigateToForgotPasswordPage() {
+  private getAllErrorMessages(): ElementArrayFinder {
```
Comment (samvloeberghs) [inline test/e2e/authentication/login/login.page.ts]: I would prefer to put private member functions on the bottom of the class. Might need to put a check on tslint for this tough..

**Comment by samvloeberghs** [comment]
@DennisJaamann thx!! the annotations look very nice. 
can you check the comments please? Once those fix I'll see if I can succesfully merge! :)

File: test/e2e/authentication/authentication.page.ts (hunk 1 above)
Comment (DennisJaamann) [inline test/e2e/authentication/authentication.page.ts]: Hi Sam, yes indeed you could. But you could also omit the return type or type it as any.

To me, this particular notation really highlights how you intended the method to be used.

**Comment by samvloeberghs** [comment]
merged, thx @DennisJaamann

**Pull request merged**